spec/dummy/config/initializers/serialize_positional_to_kwargs_bridge.rb

```diff
@@ -0,0 +1,19 @@
+module Dummy
+  module SerializePositionalToKwargsBridge
+    def serialize(*args, **options)
```

Comment:
This is the largest change in rails 7.1+ for this repository.  Other than the "coder" change above, there isn't an option to default the `type` so what I did was accepted the new interface: kwargs for type and built a bridge to convert kwargs back to positional arguments when run with rails 7.0.